stable/drupal/README.md

```diff
@@ -55,10 +55,10 @@ The following tables lists the configurable parameters of the Drupal chart and t
 | `mariadb.mariadbRootPassword`     | MariaDB admin password                | `nil`                                                     |
 | `serviceType`                     | Kubernetes Service type               | `LoadBalancer`                                            |
 | `persistence.enabled`             | Enable persistence using PVC          | `true`                                                    |
-| `persistence.apache.storageClass` | PVC Storage Class for Apache volume   | `nil` (uses alpha storage class annotation)               |
+| `persistence.apache.storageClass` | PVC Storage Class for Apache volume   | `nil` (uses beta storage class annotation)               |
```

Comment:
Btw, I think this might be an important distinction as a value of `nil` could imply the annotation is left out entirely and a default storage class (if one exists) is used, but this is not currently the case.